tox.ini

```diff
@@ -61,18 +61,10 @@ deps =
 setenv =
     PYTHONPATH =
 
-[testenv:coveralls]
-basepython = {[testenv:cover]basepython}
-commands =
-    pip install gcloud[grpc]
-    {[testenv]covercmd}
-    coveralls
-deps =
-    {[testenv:cover]deps}
-    coveralls
-setenv =
-    PYTHONPATH =
-passenv = {[testenv:system-tests]passenv}
+[testenv:codecov]
+passenv = CI TRAVIS_BUILD_ID TRAVIS TRAVIS_BRANCH TRAVIS_JOB_NUMBER TRAVIS_PULL_REQUEST TRAVIS_JOB_ID TRAVIS_REPO_SLUG TRAVIS_COMMIT
+deps = codecov>=1.4.0
+commands = codecov -e TOXENV
```

Comment:
That is just an example. From the CLI tool help:

```
  --env [ENV [ENV ...]], -e [ENV [ENV ...]]
                        Store environment variables to help distinguish CI
                        builds. Example: http://bit.ly/1ElohCu
```

This means `-e` is optional. We only run coverage in Python 2.7 so we don't need to distinguish.